lib/Field/FieldLeft.js

```diff
@@ -0,0 +1,27 @@
+import React, { useContext } from 'react';
+import Col from 'react-bootstrap/lib/Col';
+import { node, string } from 'prop-types';
+import { FieldInStructureEditModeContext } from './FieldInStructureEditModeProvider';
+
+export const FieldLeft = ({ children, className }) => {
```

Comment:
🔴 What was the thinking around `FieldLeft`? Naming things is hard but left doesn't tell us anything about what it should house. Here are some ideas;

- `FieldData` (funnily enough this has a class name for this)
- `FieldInformation`
- `FieldAside`

Another point is we have right to left settings in some usages. Does `FieldLeft` make sense in those cases?

Reply:
Dismiss this feedback. Too much effort to name things 😂